src/components/views/settings/ProfileSettings.js

```diff
@@ -52,13 +53,18 @@ export default class ProfileSettings extends React.Component {
         this._avatarUpload = createRef();
     }
 
-    _uploadAvatar = (e) => {
-        e.stopPropagation();
-        e.preventDefault();
-
+    _uploadAvatar = () => {
         this._avatarUpload.current.click();
     };
 
+    _removeAvatar = () => {
+        this.setState({
+            avatarUrl: null,
+            avatarFile: null,
+            enableProfileSave: true,
+        });
```

Comment:
needs a profileSave() call?

Reply:
Nope, profile is saved when you click the Save button, thus `enableProfileSave`

Reply:
ah right, this is one of the areas where we don't live-apply changes for some reason.